### ISSUE TITLE
Should reply to "reply-to" field, not the "from" field

### ISSUE THREAD

**Issue by dbe** [eladnava/batch-reply-for-gmail#1]
Tried using this to respond to the hundreds of recruitment emails from linkedin, however those emails are from a dead end email address, but have the correct "reply-to" email in the headers.

**Comment by eladnava** [comment]
This is a good idea. Do you want to change it and submit a pull request?

**Comment by dbe** [comment]
I just went in to see if its possible.

Unfortunately it looks like gmail doesn't expose the reply-to address in the list view. In order to fetch the reply-to you would most likely need to fully change the architecture of this plugin :(

**Comment by eladnava** [comment]
Indeed, it appears that way. I'll check if there's another way to get this done.
